.github/policies/resourceManagement.yml

```
@@ -0,0 +1,518 @@
+            hasLabel:
+              label: Type:DCR
+        - not:
+            hasLabel:
+              label: Type:Bug
+        - not:
+            hasLabel:
+              label: Type:DeveloperDocs
+        - not:
+            hasLabel:
+              label: Type:DataAnalysis
+        - not:
+            hasLabel:
+              label: Type:Learning
+        - not:
+            hasLabel:
+              label: Type:Tracking
+        - not:
+            hasLabel:
+              label: Type:Engineering
+        - not:
+            hasLabel:
+              label: Type:BreakingChange
+        - or:
+          - isAction:
+              action: Reopened
+          - isAction:
+              action: Assigned
+          - isAction:
+              action: Closed
+      - not:
+          hasLabel:
+            label: Epic
+      then:
+      - addReply:
+          reply: ${assignees} Issue is missing Type label, remember to add a [Type label](https://github.com/NuGet/Client.Engineering/blob/main/designs/nuget-issues-approach.md#issue-type)
+      - addLabel:
+          label: missing-required-type
+      description: '[Type Label Reminder][4-4] Remember NuGet Client member(issue assignee) to add Type label'
+    - if:
+      - payloadType: Issue_Comment
+      - isOpen
+      - isAction:
+          action: Created
+      - hasLabel:
+          label: Icebox cleanup candidate
+      - not:
+          hasLabel:
+            label: Triage:NeedsTriageDiscussion
+      then:
+      - addLabel:
+          label: Triage:NeedsTriageDiscussion
+      description: '[Close stale icebox][3-3] For "Icebox cleanup candidate" issues, if there is any comments, add "Triage:NeedsTriageDiscussion" label'
+    - if:
+      - payloadType: Pull_Request
+      - or:
+        - isAction:
+            action: Opened
+        - isAction:
+            action: Reopened
+      - activitySenderHasPermission:
+          permission: Read
+      - not:
+          isActivitySender:
+            user: 'nuget-client '
+            issueAuthor: False
+      - not:
+          hasLabel:
+            label: Community
+      then:
+      - addLabel:
+          label: Community
+      description: '[Label community PRs] [1-1] Label the community PRs'
+    - if:
+      - payloadType: Pull_Request
+      - not:
+          isAction:
+            action: Closed
+      - hasLabel:
+          label: Status:No recent activity
+      - hasLabel:
+          label: Community
+      then:
+      - removeLabel:
+          label: Status:No recent activity
+      description: '[stale community PR] [5-2] Remove "Status:No recent activity" if there is any activity.'
+    - if:
+      - payloadType: Issue_Comment
+      - hasLabel:
+          label: Status:No recent activity
+      - hasLabel:
+          label: Community
+      then:
+      - removeLabel:
+          label: Status:No recent activity
+      description: '[stale community PR] [5-3] Remove "Status:No recent activity" if there is any comment.'

```

The previous configuration was limited to pull requests via the `"eventType": "pull_request"` configuration. This will bring back that condition. /cc @JohannesLampel

```suggestion
    - if:
      - payloadType: Issue_Comment
      - isPullRequest
      - hasLabel:
          label: Status:No recent activity
      - hasLabel:
          label: Community
      then:
      - removeLabel:
          label: Status:No recent activity
      description: '[stale community PR] [5-3] Remove "Status:No recent activity" if there is any comment.'
```